### PR TITLE
Read config from already-parsed global config, not a JSON file.

### PR DESCRIPTION
**Pull request by nhooey**
Supplying a filename to read and parse the JSON out of is not the intent
of the `--jsonconfig` parameter. It's supposed to read from the global
Sensu configuration that has already been read, JSON-parsed, and merged
from `/etc/sensu/conf.d` or similar.

See the following plugins for this convention:
- [sensu-plugins-pagerduty](https://github.com/sensu-plugins/sensu-plugins-pagerduty/blob/master/bin/handler-pagerduty.rb#L35)
- [sensu-plugins-slack](https://github.com/sensu-plugins/sensu-plugins-slack/blob/master/bin/handler-slack.rb#L82)
- [sensu-plugins-hipchat](https://github.com/sensu-plugins/sensu-plugins-hipchat/blob/master/bin/handler-hipchat.rb#L42)

### CODE REVIEW

File: bin/handler-opsgenie.rb [hunk 1]
```diff
@@ -12,15 +12,15 @@
 class Opsgenie < Sensu::Handler
   option :json_config,
          description: 'Configuration name',
-         short: '-j JSONCONFIG',
-         long: '--json JSONCONFIG',
-         default: 'opsgenie.json'
+         short: '-j <config-name>',
```
Comment (analytically) [inline bin/handler-opsgenie.rb:15]: what's the reason for this change?

**Comment by analytically** [comment]
Can you bump the major version and add an entry in the changelog and update the README?

**Comment by analytically** [comment]
Bump.

**Comment by nhooey** [comment]
I'll be done this by the end of the weekend.

File: bin/handler-opsgenie.rb (hunk 1 above)
Comment (nhooey) [inline bin/handler-opsgenie.rb:15]: From the commit log:

```
Read config from already-parsed global config, not a JSON file.

Supplying a filename to read and parse the JSON out of is not the intent
of the `--jsonconfig` parameter. It's supposed to read from the global
Sensu configuration that has already been read, JSON-parsed, and merged
from `/etc/sensu/conf.d` or similar.
```

**Comment by nhooey** [comment]
I'm not sure what to put the in the `README`. Let me know what you're thinking.

Otherwise it's ready for release.

**Comment by nhooey** [comment]
@analytically Could you cut a new release of this?

**Comment by nhooey** [comment]
@sstarcher Could you make a new release so this pull request can be included?

**Comment by sstarcher** [comment]
@nhooey tag @mattyjones  for that one

**Comment by nhooey** [comment]
@mattyjones Could you make a new release so this pull request can be included?

**Comment by analytically** [comment]
In 1.0.0.
